src/redux/actions/internalActions.js

```diff
@@ -242,6 +242,10 @@ export const setMouseWheelZoom = (enableMouseWheelZoom = true) => ({
   type: 'SET_MOUSE_WHEEL_ZOOM',
   payload: { enableMouseWheelZoom },
 });
+export const setTextEdit = (enableTextEdit = true) => ({
```

Comment:
Similar to my comment at https://github.com/PDFTron/webviewer-ui/pull/824#discussion_r474273242.

Reply:
Same as the other comment, removed TextEdit reducer and actions